src/cartridges/int_adyen_SFRA/cartridge/client/default/js/expressPaymentMethodsVisibility.js

```diff
@@ -0,0 +1,20 @@
+async function handleExpressPaymentsVisibility() {
+  const { expressMethodsOrder } = window;
+  if (expressMethodsOrder) {
+    const sortOrder = expressMethodsOrder.split(',');
+    const container = document.getElementById('express-container');
+    let toSort = container.children;
+    toSort = Array.prototype.slice.call(toSort, 0);
+    toSort.sort(
+      (a, b) =>
+        sortOrder.indexOf(a.dataset.method) -
+        sortOrder.indexOf(b.dataset.method),
+    );
+    container.innerHTML = '';
+    for (let i = 0; i < toSort.length; i += 1) {
```

Comment:
we could use "toSort.map" in place of for loop here.

Reply:
using spread operator here 👍